### PR TITLE
Add property number

### PR DESCRIPTION
**Pull request by KirthiNair**

### CODE REVIEW

File: src/shared/modules/itemEditor/itemEditorDuck.js [hunk 1]
```diff
@@ -85,6 +85,24 @@ export const handleFetchDataEpic = (action$, store) =>
       })
   })
 
+/**
+ * This function returns the cypher compatible value of the received value.
+ */
+let convertedValue = ''
```
Comment (rahul-tarafdar-synerzip) [inline src/shared/modules/itemEditor/itemEditorDuck.js]: get this variable inside getCypherCompatibleValue() method

File: src/shared/modules/itemEditor/itemEditorDuck.js [hunk 2]
```diff
@@ -96,10 +114,11 @@ export const handleEditEntityEpic = (action$, store) =>
     let cmd
     switch (action.editType) {
       case 'create':
+        getCypherCompatibleValue(action)
         if (action.entityType === 'nodeProperty') {
           cmd = `MATCH (a)
         WHERE ID(a) = ${action.editPayload.id}
-        SET a.${action.editPayload.key} = '${action.editPayload.value}'
+        SET a.${action.editPayload.key} = ${convertedValue}
```
Comment (rahul-tarafdar-synerzip) [inline src/shared/modules/itemEditor/itemEditorDuck.js]: call getCypherCompatibleValue() method to get value in here

**Review by rahul-tarafdar-synerzip — CHANGES_REQUESTED** [review]

File: src/shared/modules/itemEditor/itemEditorDuck.js (hunk 2 above)
Comment (KirthiNair) [reply src/shared/modules/itemEditor/itemEditorDuck.js]: done

File: src/shared/modules/itemEditor/itemEditorDuck.js (hunk 1 above)
Comment (KirthiNair) [reply src/shared/modules/itemEditor/itemEditorDuck.js]: done

**Review by rahul-tarafdar-synerzip — APPROVED** [review]

**Pull request merged**